src/services/item/plugins/importExport/service.ts

```diff
@@ -33,6 +33,17 @@ import {
 import { UnexpectedExportError } from './errors';
 import { buildTextContent, getFilenameFromItem } from './utils';
 
+export type GraaspExportItem = {
+  id: string;
+  name: string;
+  type: ItemTypeUnion;
+  description: string | null;
+  settings: ItemSettings;
+  thumbnailFilename?: string;
+  children?: GraaspExportItem[];
+  mimetype?: string;
```

Comment:
So these are different if it's for a folder or a file right, maybe let's have some comments.